example/src/main/java/com/stripe/example/activity/LaunchPaymentSheetCompleteActivity.kt

```diff
@@ -46,34 +57,49 @@ internal class LaunchPaymentSheetCompleteActivity : BasePaymentSheetActivity() {
             it.fold(
                 onSuccess = { json ->
                     val clientSecret = json.getString("secret")
+                    viewModel.inProgress.postValue(false)
```

Comment:
I think I didn't add this intentionally because, while there are no network requests in progress, we're about to fire a new network request. This is fine though.

Reply:
This is code moved from `onPaymentIntent` which is now inline here and below on `createSetupIntent`. We were posting false then. I'll revisit as I finish implementation.